layersvt/json/VkLayer_api_dump.json.in

```diff
@@ -133,183 +133,191 @@
+              "type": "BOOL",
+              "default": false,
+              "settings": [
+                {
+                  "key": "log_filename",
+                  "env": "VK_APIDUMP_LOG_FILENAME",
+                  "label": "Log Filename",
+                  "description": "Specifies the file to dump to when output files are enabled",
+                  "type": "SAVE_FILE",
+                  "filter": "*.txt,*.html,*.json",
+                  "default": "stdout",
+                  "dependence": {
+                    "mode": "ALL",
                     "settings": [
-                        {
-                            "key": "log_filename",
-                            "env": "VK_APIDUMP_LOG_FILENAME",
-                            "label": "Log Filename",
-                            "description": "Specifies the file to dump to when output files are enabled",
-                            "type": "SAVE_FILE",
-                            "filter": "*.txt,*.html,*.json",
-                            "default": "stdout",
-                            "dependence": {
-                                "mode": "ALL",
-                                "settings": [
-                                    {
-                                        "key": "file",
-                                        "value": true
-                                    }
-                                ]
-                            }
-                        }
+                      {
+                        "key": "file",
+                        "value": true
+                      }
                     ]
-                },
-                {
-                    "key": "flush",
-                    "env": "VK_APIDUMP_FLUSH",
-                    "label": "Log Flush After Write",
-                    "description": "Setting this to true causes IO to be flushed after each API call that is written",
-                    "type": "BOOL",
-                    "default": true
-                },
-                {
-                    "key": "name_size",
-                    "label": "Name Size",
-                    "description": "The number of characters the name of a variable should consume, assuming more are not required",
-                    "type": "INT",
-                    "default": 32,
-                    "range": {
-                        "min": 1
-                    },
-                    "unit": "chars"
-                },
+                  }
+                }
+              ]
+            },
+            {
+              "key": "flush",
+              "env": "VK_APIDUMP_FLUSH",
+              "label": "Log Flush After Write",
+              "description": "Setting this to true causes IO to be flushed after each API call that is written",
+              "type": "BOOL",
+              "default": true
+            },
+            {
+              "key": "pre_dump",
```

Comment:
Considering this is a text mode only setting, the settings should setup a "dependence" element requiring `output_format` to be set with the `text` value. This manifest contains an exemple of "dependence" usage.